src/lib/instrument-listen.js

```diff
@@ -3,24 +3,24 @@
  */
 // @ts-nocheck
 
-const raven = require('@financial-times/n-raven');
 const metrics = require('next-metrics');
 const nLogger = require('@financial-times/n-logger').default;
 const http = require('http');
 const https = require('https');
 const denodeify = require('denodeify');
 const path = require('path');
+const {STATUS_CODES} = http;
```

Comment:
TIL

Reply:
Yep the number of times I see people including a library for this! It's right there in `http` 😁